src/AnnotationsReader/ExtendedAnnotationsReader.ts

```diff
@@ -3,12 +3,14 @@ import ts from "typescript";
 import { Annotations } from "../Type/AnnotatedType";
 import { symbolAtNode } from "../Utils/symbolAtNode";
 import { BasicAnnotationsReader } from "./BasicAnnotationsReader";
+import { getJsDocTagText } from "../Utils/getJsDoc";
+import { AnnotationsReader } from "../Interfaces/AnnotationsReader";
+import { BaseType } from "../Type/BaseType";
+import { ObjectType } from "../Type/ObjectType";
+import { StringMap } from "../Utils/StringMap";
 
-export class ExtendedAnnotationsReader extends BasicAnnotationsReader {
-    public constructor(
-        private typeChecker: ts.TypeChecker,
-        extraTags?: Set<string>
-    ) {
+export class ExtendedAnnotationsReader extends BasicAnnotationsReader implements AnnotationsReader {
+    public constructor(private typeChecker: ts.TypeChecker, extraTags?: Set<string>) {
```

Comment:
these style changes are made by `prettier`.

This repo's root has an prettier config file, so my vscode use prettier to format every file